.travis.yml

```diff
@@ -3,3 +3,5 @@ python: 3.5
 cache: pip
 before_install: pip install -U pre-commit
 script: pre-commit run --all-files
+notifications:
+  email: false
```

Comment:
You could configure email filtering. AFAIK this will prevent travis from sending notifications other users might want to receive, which is weird. I believe there should be some other way of disabling it on per user/per repo basis.